ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -35,12 +35,14 @@ function pre-ocp-66839(){
         echo "Failed to extract manifests!"
         return 1
     fi
-    # There should not be any cap annotation in all extracted manifests 
-    curCap=$(grep -rh "capability.openshift.io/name:" "${manifestsDir}"|awk -F": " '{print $NF}'|sort -u)
-    if [[ "${curCap}" != "" ]]; then
-        echo "Caps in extracted manifests found: ${curCap}, but expected nothing"
+
+    # There should be only enabled cap annotaion in all extracted manifests
+    curCap=$(grep -rh "capability.openshift.io/name:" "${manifestsDir}"|awk -F": " '{print $NF}'|sort -u|xargs)
+    if [[ "${curCap}" != "${ADDITIONAL_ENABLED_CAPABILITIES}" ]]; then
```

Comment:
Per the logic, `ADDITIONAL_ENABLED_CAPABILITIES` has to be in a sorted order for multiple caps, while it is also used in fresh install, while in fresh install, there is no a rule to limit its value has to be in a sorted order. 

To make the codes be robust, maybe we need to sort `ADDITIONAL_ENABLED_CAPABILITIES` before the compare? Similar comments on https://github.com/openshift/release/pull/44820/files#diff-bcec571809d92fb0e609baf8907886244e300b0f2aa08759cc9f6cac2154a225R39

Reply:
good point, updated